apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -813,12 +831,22 @@ export class LemonadePipeline implements BasePipeline {
         });
       }
 
+      const ticketPCDs = await Promise.all(
```

Comment:
name of this var is slightly too broad now

Reply:
Moving the handling of the PODTicketPCDs fixes this, so it's no longer part of a pair with podTicketPCDs but is clearly the superset of it.